draft-ietf-intarea-provisioning-domains.md

```diff
@@ -119,7 +119,10 @@ the PvD ID Router Advertisement option which, when present, associates
 the PvD ID with all the information present in the Router Advertisement
 as well as any configuration object, such as addresses, derived from
 it. The PVD ID Router Advertisement option may also contain a set of
-other RA options. These options are only visible to 'PvD-aware' hosts.
+other RA options, along with an optional inner Router Advertisement
+message header. These options and optional header are only visible
```

Comment:
s/optional header/optional inner RA header/

Just to be clear (albeit a little redundant...)